openwisp_radius/tests/test_models.py

```diff
@@ -588,7 +588,7 @@ def setUp(self):
 
     def _download_csv_file_status(self, status_code):
         response = self.client.get(
-            reverse('radius:serve_private_file', args=[self.csvfile],)
+            reverse('radius_privstore:serve_private_file', args=[self.csvfile],)
```

Comment:
why is this kind of thing needed?
How does this play with the already deployed instances? For example, how does this change play with the work we did in ansible-openwisp2?

Reply:
So, previously we were using the same namespace for multiple apps.
This is not ideal. Django usually wants separate namespaces for each application. All other openwisp applications have separated namespaces, somehow, radius was done differently and all the apps were thrown inside `radius` namespace.

While this was working so far, it's not working now that I need to change `urlconf` for a url and it can't find urls with previous structure.

So, I've changed to more idiomatic django code and now `urlconf` can read the unique namespaces for different apps.

No change shall be required in ansible-openwisp2.

Reply:
The problem Ajay is that these subfolders in openwisp radius are not django apps.
OpenWISP RADIUS is a single app with different features (API, social login, saml login, etc), but it's only one django app, therefore I don't think we should add more namespaces because it would be confusing.
Can you please give pratical examples of how the single namespace is affecting you negatively so we can look for solutions?

At the moment to me it's still not clear why we need to add new namespaces.